app/assets/javascripts/utils/api.js

```diff
@@ -36,7 +36,7 @@ const API = {
     })
       .then(res => {
         if (res.ok) {
-          return res.json();
+          return Promise.resolve({ statusText: res.statusText });
```

Comment:
previously it was throwing an error (shown below) :

[Demo1.webm](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/119471995/e0773dce-9de7-42f9-a370-6c140ba4ad98)

This error message ("Unexpected token '<', "<!DOCTYPE "... is not valid JSON"), indicates that the parser encountered an unexpected '<' character, which is usually indicative of HTML content, not JSON.
So, then i replaced that line with Promise.resolve({ statusText: res.statusText }); as this will be returned when status of response is okay.

After :
[Demo2.webm](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/119471995/eb9f14a2-fe26-4afe-8920-4a3f3cc9ac8e)